pkg/guestagent/guestagent_linux.go

```diff
@@ -326,6 +326,10 @@ func (a *agent) fixSystemTimeSkew() {
 	for {
 		ticker := time.NewTicker(10 * time.Second)
 		for now := range ticker.C {
+			if !timesync.DeviceExists() {
+				logrus.Infof("fixSystemTimeSkew: device does not exist")
```

Comment:
```suggestion
				logrus.Info("fixSystemTimeSkew: /dev/rtc does not exist")
```

Reply:
now it says `level=warning msg="fixSystemTimeSkew: error: stat /dev/rtc: no such file or directory"` (once)